docs(terminus): more explicit guide for HTTPHealthIndicator (#2302)
nestjs/docs.nestjs.com: content/recipes/terminus.md

Once we have installed `@nestjs/terminus`, imported our `TerminusModule` and created a new controller, we are ready to create a health check.

> The `HTTPHealthIndicator` makes use of the `@nestjs/axios` package. In order to use it, make sure you have the following package installed:

Reviewer: ```suggestion
The `HTTPHealthIndicator` requires the `@nestjs/axios` package. To use it, make sure you have the following package installed:
```

```bash
$ npm i --save @nestjs/axios
```

> No we can setup our `HealthController`:

Reviewer: ```suggestion
Now we can setup our `HealthController`:
```
